[Fastlane.Swift] Badly generated file Plugins.swift, sanitize comments.

<!-- Thanks for helping fastlane! Before you submit your issue, please make sure you followed our checklist and check the appropriate boxes by putting an x in the [ ]: [x] -->

### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
<!-- Please include what's happening, expected behavior, and any relevant code samples -->
When using Swift version of Fastlane, `Plugins.swift` file is generated wrong.
If plugin docs contains unfortunate combination of `/` and `*` characters it results in `Unterminated '/*' comment` error in Xcode for `FastlaneSwiftRunner.xcodeproj`.
It happens for example for `fastlane-plugin-lizard` https://github.com/mosesliao/fastlane-plugin-lizard.

Problematic docs:
```
exclude: Exclude files that match this pattern. * matches everything, ? matches any single character, "./folder/*" exclude everything in the folder recursively. Multiple patterns can be specified. Don't forget to add "" around the pattern
```

Please fix this issue or provide workaround.

##### Command executed
`fastlane update_plugins`
`fastlane example`

##### Complete output when running fastlane, including the stack trace and command used
<!-- 
You can use `--capture_output` as the last command line argument for many commands to get that collected for you. Otherwise, please do it manually.
Caution: The output of `--capture_output` could contain sensitive data such as application ids, certificate ids, or email addresses. Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue 
-->

`fastlane update_plugins`
<details>
<pre>
[17:48:16]: fastlane detected a Gemfile in the current directory
[17:48:16]: However, it seems like you didn't use `bundle exec`
[17:48:16]: To launch fastlane faster, please use
[17:48:16]: 
[17:48:16]: $ bundle exec fastlane update_plugins
[17:48:16]: 
[17:48:16]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
+-------------------------------------------+---------+-----------------------------------------------------------+
|                                                  Used plugins                                                   |
+-------------------------------------------+---------+-----------------------------------------------------------+
| Plugin                                    | Version | Action                                                    |
+-------------------------------------------+---------+-----------------------------------------------------------+
| fastlane-plugin-lizard                    | 1.3.3   | lizard                                                    |
| fastlane-plugin-firebase_app_distribution | 0.2.5   | firebase_app_distribution_login firebase_app_distribution |
| fastlane-plugin-teams                     | 0.2.0   | teams                                                     |
+-------------------------------------------+---------+-----------------------------------------------------------+

Updating plugin dependencies...
[✔] 🚀 
+-------------------------------------------+---------+-----------------------------------------------------------+
|                                                  Used plugins                                                   |
+-------------------------------------------+---------+-----------------------------------------------------------+
| Plugin                                    | Version | Action                                                    |
+-------------------------------------------+---------+-----------------------------------------------------------+
| fastlane-plugin-lizard                    | 1.3.3   | lizard                                                    |
| fastlane-plugin-firebase_app_distribution | 0.2.5   | firebase_app_distribution_login firebase_app_distribution |
| fastlane-plugin-teams                     | 0.2.0   | teams                                                     |
+-------------------------------------------+---------+-----------------------------------------------------------+

[17:48:26]: <redacted>/<project_dir>/fastlane/swift/Actions.swift
[17:48:26]: <redacted>/<project_dir>/fastlane/swift/Plugins.swift
Successfully updated plugins
</pre>
</details>

`fastlane example`
<details>
<pre>
[✔] 🚀 
[17:50:34]: fastlane detected a Gemfile in the current directory
[17:50:34]: However, it seems like you didn't use `bundle exec`
[17:50:34]: To launch fastlane faster, please use
[17:50:34]: 
[17:50:34]: $ bundle exec fastlane example
[17:50:34]: 
[17:50:34]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
+-------------------------------------------+---------+-----------------------------------------------------------+
|                                                  Used plugins                                                   |
+-------------------------------------------+---------+-----------------------------------------------------------+
| Plugin                                    | Version | Action                                                    |
+-------------------------------------------+---------+-----------------------------------------------------------+
| fastlane-plugin-lizard                    | 1.3.3   | lizard                                                    |
| fastlane-plugin-firebase_app_distribution | 0.2.5   | firebase_app_distribution_login firebase_app_distribution |
| fastlane-plugin-teams                     | 0.2.0   | teams                                                     |
+-------------------------------------------+---------+-----------------------------------------------------------+

[17:50:35]: loading manifest: /usr/local/Cellar/fastlane/2.180.1_1/libexec/gems/fastlane-2.180.1/fastlane/swift/upgrade_manifest.json
[17:50:35]: FastlaneSwiftRunner project is up-to-date
[17:50:35]: Resolving Swift Package Manager dependencies...
[17:50:35]: $ xcodebuild -resolvePackageDependencies -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
[17:50:36]: ▸ Command line invocation:
[17:50:36]: ▸     "/Applications/Xcode 12.2.app/Contents/Developer/usr/bin/xcodebuild" -resolvePackageDependencies -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
[17:50:37]: ▸ resolved source packages: 
[17:50:37]: $ xcodebuild -showBuildSettings -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
[17:50:38]: $ set -o pipefail && xcodebuild -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj -destination 'generic/platform=macOS' build | tee <redacted>/Library/Logs/gym/App-FastlaneRunner.log | xcpretty
▸ Compiling StaticAnalysis.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
 ^


▸ Compiling LaneFileProtocol.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^


▸ Compiling Fastfile.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^



❌  /<project_dir>/fastlane/Fastfile.swift:13:5: cannot find 'lizard' in scope

    lizard(
^


▸ Compiling Actions.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
    ^~~~~~


▸ Compiling SocketClient.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^


▸ Compiling Runner.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^


▸ Compiling main.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^


▸ Compiling Plugins.swift

❌  /<project_dir>/fastlane/swift/Plugins.swift:142:2: unterminated '/*' comment

}
^


** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/core/StaticAnalysis.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/LaneFileProtocol.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/Fastfile.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/Actions.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/SocketClient.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/Runner.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/main.swift
	CompileSwift normal x86_64 <redacted>/<project_dir>/fastlane/swift/Plugins.swift
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(9 failures)
[17:50:41]: Exit status: 65
+-----------+---------+
|    Lane Context     |
+-----------+---------+
| LANE_NAME | example |
+-----------+---------+
[17:50:41]: Exit status: 65
[17:50:41]: fastlane finished with errors

[!] Exit status: 65

</pre>
</details>

### Environment

<!-- 
Please run `fastlane env` and copy the output below. This will help us help you.
If you used the `--capture_output` option, please remove this block as it is already included there. 
-->

<details>
  <pre> <details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                     |
| --------------------------- | --------------------------------------------------------- |
| OS                          | 10.15.7                                                   |
| Ruby                        | 2.7.3                                                     |
| Bundler?                    | false                                                     |
| Git                         | git version 2.24.3 (Apple Git-128)                        |
| Installation Source         | /usr/local/Cellar/fastlane/2.180.1_1/libexec/bin/fastlane |
| Host                        | Mac OS X 10.15.7 (19H2)                                   |
| Ruby Lib Dir                | /usr/local/Cellar/ruby@2.7/2.7.3/lib                      |
| OpenSSL Version             | OpenSSL 1.1.1k  25 Mar 2021                               |
| Is contained                | false                                                     |
| Is homebrew                 | true                                                      |
| Is installed via Fabric.app | false                                                     |
| Xcode Path                  | /Applications/Xcode 12.2.app/Contents/Developer/          |
| Xcode Version               | 12.2                                                      |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | pl_PL.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile.swift`</summary>

```ruby
// This file contains the fastlane.tools configuration
// You can find the documentation at https://docs.fastlane.tools
//
// For a list of all available actions, check out
//
//     https://docs.fastlane.tools/actions
//

import Foundation

class Fastfile: LaneFile {
    func exampleLane() {
    lizard(
        sourceFolder: <redacted>,
        language: "swift",
        exportType: "xml",
        reportFile: "reports/lizard-report.xml")
    }
}


```
</details>

**No Appfile found**


### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.180.1 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status |
| ----------------------------------------- | ------- | ------------- |
| fastlane-plugin-lizard                    | 1.3.3   | ✅ Up-To-Date  |
| fastlane-plugin-firebase_app_distribution | 0.2.5   | ✅ Up-To-Date  |
| fastlane-plugin-teams                     | 0.2.0   | ✅ Up-To-Date  |


<details><summary><b>Loaded gems</b></summary>

| Gem                                       | Version      |
| ----------------------------------------- | ------------ |
| did_you_mean                              | 1.4.0        |
| slack-notifier                            | 2.3.2        |
| rouge                                     | 2.0.7        |
| xcpretty                                  | 0.3.0        |
| terminal-notifier                         | 2.0.0        |
| unicode-display_width                     | 1.7.0        |
| terminal-table                            | 1.8.0        |
| public_suffix                             | 4.0.6        |
| addressable                               | 2.7.0        |
| multipart-post                            | 2.0.0        |
| word_wrap                                 | 1.0.0        |
| tty-screen                                | 0.8.1        |
| tty-cursor                                | 0.7.1        |
| tty-spinner                               | 0.9.3        |
| artifactory                               | 3.0.15       |
| colored                                   | 1.2          |
| highline                                  | 1.7.10       |
| commander-fastlane                        | 4.4.6        |
| unf_ext                                   | 0.0.7.7      |
| unf                                       | 0.1.4        |
| domain_name                               | 0.5.20190701 |
| http-cookie                               | 1.0.3        |
| faraday-cookie_jar                        | 0.0.7        |
| faraday_middleware                        | 1.0.0        |
| gh_inspector                              | 1.1.3        |
| rubyzip                                   | 2.3.0        |
| security                                  | 0.1.3        |
| dotenv                                    | 2.7.6        |
| simctl                                    | 1.6.8        |
| jwt                                       | 2.2.2        |
| retriable                                 | 3.1.2        |
| httpclient                                | 2.8.3        |
| google-api-client                         | 0.38.0       |
| uri                                       | 0.10.0       |
| nanaimo                                   | 0.3.0        |
| colored2                                  | 3.1.2        |
| claide                                    | 1.0.3        |
| CFPropertyList                            | 3.0.3        |
| atomos                                    | 0.1.3        |
| xcodeproj                                 | 1.19.0       |
| plist                                     | 3.6.0        |
| babosa                                    | 1.0.4        |
| excon                                     | 0.80.0       |
| ruby2_keywords                            | 0.0.4        |
| faraday-net_http                          | 1.0.1        |
| faraday                                   | 1.3.0        |
| fastimage                                 | 2.2.3        |
| json                                      | 2.5.1        |
| mini_magick                               | 4.11.0       |
| naturally                                 | 2.2.1        |
| xcpretty-travis-formatter                 | 1.0.1        |
| bundler                                   | 2.2.16       |
| trailblazer-option                        | 0.1.1        |
| declarative                               | 0.0.20       |
| uber                                      | 0.1.0        |
| representable                             | 3.1.1        |
| mini_mime                                 | 1.1.0        |
| multi_json                                | 1.15.0       |
| signet                                    | 0.15.0       |
| os                                        | 1.1.1        |
| memoist                                   | 0.16.2       |
| googleauth                                | 0.16.1       |
| rake                                      | 13.0.3       |
| digest-crc                                | 0.6.3        |
| webrick                                   | 1.7.0        |
| rexml                                     | 3.2.5        |
| google-apis-core                          | 0.3.0        |
| google-apis-storage_v1                    | 0.3.0        |
| google-apis-iamcredentials_v1             | 0.3.0        |
| google-cloud-errors                       | 1.1.0        |
| google-cloud-env                          | 1.5.0        |
| google-cloud-core                         | 1.6.0        |
| google-cloud-storage                      | 1.31.0       |
| emoji_regex                               | 3.2.2        |
| aws-eventstream                           | 1.1.1        |
| aws-sigv4                                 | 1.2.3        |
| aws-partitions                            | 1.444.0      |
| jmespath                                  | 1.4.0        |
| aws-sdk-core                              | 3.114.0      |
| aws-sdk-kms                               | 1.43.0       |
| aws-sdk-s3                                | 1.93.1       |
| forwardable                               | 1.3.1        |
| logger                                    | 1.4.2        |
| cgi                                       | 0.1.0        |
| date                                      | 3.0.0        |
| timeout                                   | 0.1.0        |
| stringio                                  | 0.1.0        |
| zlib                                      | 1.1.0        |
| ipaddr                                    | 1.2.2        |
| openssl                                   | 2.1.2        |
| ostruct                                   | 0.2.0        |
| strscan                                   | 1.0.3        |
| delegate                                  | 0.1.0        |
| fileutils                                 | 1.4.1        |
| io-console                                | 0.5.6        |
| open3                                     | 0.1.0        |
| singleton                                 | 0.1.0        |
| nokogiri                                  | 1.11.3       |
| racc                                      | 1.5.2        |
| yaml                                      | 0.1.0        |
| psych                                     | 3.1.0        |
| mutex_m                                   | 0.1.0        |
| coderay                                   | 1.1.3        |
| method_source                             | 1.0.0        |
| pry                                       | 0.14.1       |
| fastlane-plugin-lizard                    | 1.3.3        |
| fastlane-plugin-firebase_app_distribution | 0.2.5        |
| pstore                                    | 0.1.0        |
| fastlane-plugin-teams                     | 0.2.0        |
</details>


*generated on:* **2021-04-15**
</details> </pre>
</details>

--- a/fastlane/lib/fastlane/swift_fastlane_function.rb
+++ b/fastlane/lib/fastlane/swift_fastlane_function.rb
@@ -232,7 +232,7 @@ module Fastlane
       # Adds newlines between each documentation element.
       documentation = documentation_elements.flat_map { |element| [element, separator] }.tap(&:pop).join("\n")
 
-      return "/**\n#{documentation}\n*/\n"
+      return "/**\n#{documentation.gsub('/*', '/\\*')}\n*/\n"
     end
 
     def swift_parameter_documentation

